fmriprep/workflows/anatomical.py

```diff
@@ -542,6 +543,7 @@ def init_surface_recon_wf(omp_nthreads, hires, name='surface_recon_wf'):
     fs_transform = pe.Node(
         fs.Tkregister2(fsl_out='freesurfer2subT1.mat', reg_header=True),
         name='fs_transform')
+    fsl2lta = pe.Node(fs.utils.LTAConvert(out_lta=True), name='fsl2lta')
```

Comment:
I would imagine that we don't need FSL transforms because we are not chaining them? (we resample in T1 space, I assume). Is this correct?

Reply:
Answered below, thanks!

Reply:
I'm doing this right now because nipype doesn't have LTA outputs from tkregister enabled. See nipy/nipype#2217. (Though it also doesn't matter in the long run, since we drop tkregister in #733.)

Reply:
Ugh. Actually just realized I'm missing two inputs to `fsl2lta`. Hold off on merging.